assets/js/phoenix_live_view/live_socket.js

```diff
@@ -173,6 +173,8 @@ export default class LiveSocket {
 
   // public
 
+  version(){ return LV_VSN }
```

Comment:
See my note below about bundle size, as I believe this could cause the whole of the json contents to be brought into the distribution files?

Maybe with the autoparsing provided by the `require` call, esbuild would be smart enough to drop the unused contents and just inline the one string, but it should be checked.